pyproject.toml

```diff
@@ -1,13 +1,140 @@
 [build-system]
 requires = [
-    "setuptools >= 48, <60",
+    "setuptools >= 61",
     "setuptools_scm[toml] >= 4, <6",
     "setuptools_scm_git_archive",
     "milksnake",
     "wheel >= 0.29.0",
 ]
 build-backend = 'setuptools.build_meta'
 
+[project]
+name = "sourmash"
+description = "tools for comparing DNA sequences with MinHash sketches"
+readme = "README.md"
+
+authors = [
+  { name="Luiz Irber", orcid="0000-0003-4371-9659" },
+  { name="Mohamed Abuelanin", orcid="0000-0002-3419-4785" },
+  { name="Harriet Alexander", orcid="0000-0003-1308-8008" },
+  { name="Abhishek Anant", orcid="0000-0002-5751-2010" },
+  { name="Keya Barve", orcid="0000-0003-3241-2117" },
+  { name="Colton Baumler" },
+  { name="Olga Botvinnik", orcid="0000-0003-4412-7970" },
+  { name="Phillip Brooks", orcid="0000-0003-3987-244X" },
+  { name="Peter Cock", orcid="0000-0001-9513-9993" },
+  { name="Daniel Dsouza" },
+  { name="Laurent Gautier", orcid="0000-0003-0638-3391" },
+  { name="Tim Head", orcid="0000-0003-0931-3698" },
+  { name="Mahmudur Rahman Hera", orcid="0000-0002-5992-9012" },
+  { name="Hannah Eve Houts", orcid="0000-0002-7954-4793" },
+  { name="Lisa K. Johnson", orcid="0000-0002-3600-7218" },
+  { name="Fabian Klötzl", orcid="0000-0002-6930-0592" },
+  { name="David Koslicki", orcid="0000-0002-0640-954X" },
+  { name="Katrin Leinweber", orcid="0000-0001-5135-5758" },
+  { name="Marisa Lim", orcid="0000-0003-2097-8818" },
+  { name="Ricky Lim", orcid="0000-0003-1313-7076" },
+  { name="Ivan Ogasawara", orcid="0000-0001-5049-4289" },
+  { name="N. Tessa Pierce", orcid="0000-0002-2942-5331" },
+  { name="Taylor Reiter", orcid="0000-0002-7388-421X" },
+  { name="Camille Scott", orcid="0000-0001-8822-8779" },
+  { name="Andreas Sjödin", orcid="0000-0001-5350-4219" },
+  { name="Connor T. Skennerton", orcid="0000-0003-1320-4873" },
+  { name="Jason Stajich", orcid="0000-0002-7591-0020" },
+  { name="Daniel Standage", orcid="0000-0003-0342-8531" },
+  { name="S. Joshua Swamidass", orcid="0000-0003-2191-0778" },
+  { name="Connor Tiffany", orcid="0000-0001-8188-7720" },
+  { name="Pranathi Vemuri", orcid="0000-0002-5748-9594"},
+  { name="Erik Young" },
```

Comment:
Hi @erikyoung85! Can you please provide your ORCID for our records? =]

Reply:
Hey Luiz! My ORCID is 0000-0002-9195-9801. Thank you!